addons/digest/data/digest_data.xml

```diff
@@ -163,6 +170,7 @@
             </td>
         </tr>
     </table>
+    <t t-if="body" t-raw="body"/>
```

Comment:
Side question, isn't it possible to say that sanitized_body = html_sanitize(body), and then display a raw sanitized body ? It will slowdown a bit the process, but as we put the result in a mail_mail body_html that is not sanitized (text field), I think it is better to try to sanitize a bit. What do you think ?

Reply:
@tde-banana-odoo Normally it should be safe because the body come from an other QWeb template, but it might break some style no ?
(because some CSS style attributes are not allowed)

Reply:
Maybe yes. Speaking of styling, note that tips are sanitized before being sent to the digest template ...